doc/content/reference/end-device-templates/executing.md

```diff
@@ -38,6 +39,7 @@ $ cat example.json | ttn-lw-cli end-device template execute
 }
 ```
 </details>
+<br>
```

Comment:
`<br>` line breaks have been added at the end of all `details` sections, because otherwise there is very little whitespace between the collapsible section and the text below.

Reply:
I think we should fix this with styling, not with declarations in the content

Reply:
Changed to use a class instead of `<br>` tags. We cannot add this without a class because details sections before headers look ugly.